frontend/src/screens/study-landing.tsx

```diff
@@ -25,7 +25,7 @@ export default function UsersStudies() {
     const api = useStudyApi()
     const history = useHistory()
     const user = useCurrentUser()
-    const noConsent = useQueryParam('consent') == 'true'
```

Comment:
I'd say throw an error if it is neither true nor false. Maybe the error can cause a 4xx reply though.

Reply:
good point but usually it's omitted altogether.  the param's only set if the user left the survey early